source/carbonstorage.rst

```diff
@@ -42,6 +42,12 @@ Outputs of the model are expressed as Mg of carbon per pixel, and if desired, th
 
 The valuation model estimates the economic value of sequestration (not storage) as a function of the amount of carbon sequestered, the monetary value of each unit of carbon, a monetary discount rate, and the change in the value of carbon sequestration over time. Thus, valuation can only be done in the carbon model if you have a future scenario. Valuation is applied to sequestration, not storage, because market prices relate only to carbon sequestration. Discount rates are multipliers that typically reduce the value of carbon sequestration over time. The first type of discounting, the standard economic procedure of financial discounting, reflects the fact that people typically value immediate benefits more than future benefits due to uncertainty and assumed economic inflation over time. The second discount rate adjusts the social value of carbon sequestration over time. This value will change as the impact of carbon emissions on expected climate change-related damages changes. If we expect carbon sequestered today to have a greater impact on climate change mitigation than carbon sequestered in the future this second discount rate should be positive. On the other hand, if we expect carbon sequestered today to have less of an impact on climate change mitigation than carbon sequestered in the future, this second discount rate should be negative.
 
+The value of carbon sequestration over time for a given parcel *x* is:
+
+.. math:: value\_seq_x=V\frac{sequest_x}{yr\_fut-yr\_cur}\sum^{yr\_fut-yr\_cur-1}_{t=0}\frac{1}{\left(1+\frac{r}{100}\right)^t\left(1+\frac{c}{100}\right)^t}
+   :label: carbon_value
+
```

Comment:
Should the variables in this equation be defined afterwards? Like: where `V` is this and that ,`c` is that and this, etc...?

Reply:
Sure, I added variable definitions. Though, I didn't change this equation, just moved it from a different section of the text. There are many other places in the UG where variables aren't defined, or are re-used to mean different things, and fixing all that will be a bigger project.